webapp/components/survey/CategoryDetails/CategoryDetails.js

```diff
@@ -15,34 +15,30 @@ import { useAuthCanEditSurvey } from '@webapp/store/user'
 import { FormItem, Input } from '@webapp/components/form/input'
 import UploadButton from '@webapp/components/form/uploadButton'
 
-import CategoryImportSummary from './components/categoryImportSummary'
-import LevelEdit from './components/levelEdit'
+import ImportSummary from './ImportSummary'
+import LevelEdit from './LevelEdit'
```

Comment:
this component can also be renamed to LevelDetails (next PR as well)

Reply:
sure, I'll rename it in the next pull request